pkg/internal/checkup/executor/executor.go

```diff
@@ -109,6 +109,12 @@ func (e Executor) Execute(ctx context.Context, vmiUnderTestName, trafficGenVMINa
 		return status.Results{}, err
 	}
 
+	log.Printf("Clearing Trex console stats before test...")
```

Comment:
nit: Commit message:
> trex, executor: Clear stats before traffic send

The change is in executor, please consider reversing the prefix order to:
`executor, trex: `

Reply:
DONE